web/scenes/Portal/Teams/TeamId/Apps/AppId/Configuration/Advanced/page/SetupForm/graphql/server/update-setup.graphql

```diff
@@ -5,7 +5,6 @@ mutation UpdateSetup(
   $associated_domains: [String!]
   $contracts: [String!]
   $permit2_tokens: [String!]
-  $can_import_all_contacts: Boolean
```

Comment:
Why are we getting rid of this?

Reply:
I guess we might not have the logic right now

Reply:
yeah no logic so might as well